egs2/TEMPLATE/svs1/scripts/audio/format_scp.sh

```diff
@@ -57,6 +58,11 @@ if [ ! -f "${scp_dir}/wav.scp" ]; then
     echo "$0: Error: No such file: ${scp_dir}/wav.scp"
     exit 1
 fi
+if [ ! -d "xml_dump" ]; then
```

Comment:
I would not recommend having a fixed directory here. Various reasons:
- It might be broken when you want to start multiple trials with the same notation (you cannot easily change the name)
- It increases the burden or understanding, as additional information from the recipe
- We already have the scp file, no need to fix the directory here

Reply:
Fix xml dir has been deleted from format_xml_scp.sh.